Sources/WalletConnect/Storage/SessionStorage.swift

```diff
@@ -39,4 +40,11 @@ final class SessionStorage: WCSessionStorage {
     func delete(topic: String) {
         storage.delete(topic: topic)
     }
+    
+    func getAcknowledgedSessions() -> [WCSession] {
+        getAll().compactMap {
```

Comment:
I think `filter()` would be more fitting to use here, instead

Reply:
absolutely